src/printer.js

```diff
@@ -1209,13 +1209,15 @@ function genericPrintNoParens(path, options, print) {
       else parts.push("default:");
 
       if (n.consequent.find(node => node.type !== "EmptyStatement")) {
-        const cons = path.call(
-          function(consequentPath) {
-            return printStatementSequence(consequentPath, options, print);
-          },
-          "consequent"
-        );
-
+        const parent = path.getParentNode();
+        const lastCase = util.getLast(parent.cases);
+        const cons = path.call(consequentPath => {
+          return join(hardline, path.map(p => {
```

Comment:
That was definitely the second solution. I need consequentPath in this case. Sorry for the confusion, I refactored it so many times before that I didn't saw that one 👍.